src/main/java/frc/robot/subsystems/shooter/ShooterIOOutputs.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot.subsystems.shooter;
+
+import edu.wpi.first.epilogue.Logged;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+
+
+/**
+ * The outputs used by the shooter subsytem
+ */
+@Logged
+public class ShooterIOOutputs {
+
+        public double victor_speed;
+        public double talon_speed;
+
+        public Value elevationSolenoid_value;
```

Comment:
Instead of using DoubleSolenoid.Value for this input, please use a boolean. The reason we do this is that it makes logging easier to read and keeps the Subsystem logic easy and clean.